config/constants.go

```diff
@@ -3,19 +3,35 @@ package config
 import (
 	"errors"
 
+	"github.com/mitchellh/mapstructure"
+	"github.com/spf13/viper"
 	"google.golang.org/protobuf/encoding/protojson"
 )
 
 const (
-	policyKey    = "policy"
 	toKey        = "to"
 	envoyOptsKey = "_envoy_opts"
 )
 
 var (
-	errKeysMustBeStrings = errors.New("cannot convert nested map: all keys must be strings")
+	errKeysMustBeStrings          = errors.New("cannot convert nested map: all keys must be strings")
```

Comment:
Should these errors be prepended with the package name (e.g. `pomerium/config:`)

Reply:
they're eventually wrapped by other messages that do mention `config`, thus I guess it'll be excessive. 

my thinking here is 
1. start collecting all errors into constants so it'll be easy to review errors user may see 
2. make every user facing error having a short URL pointing to KB article regarding problem root cause and suggested actions to fix

Reply:
makes sense to me.